samples/boost-sample/exclude.def

```diff
@@ -19,7 +19,6 @@ src/inspect/unnamed_namespace_check.hpp:28:  Do not indent within a namespace.
 src/inspect/unnamed_namespace_check.hpp:28:  { should almost always be at the end of the previous line  [whitespace/braces] [4]
 src/inspect/unnamed_namespace_check.hpp:29:  Do not indent within a namespace.  [whitespace/indent_namespace] [4]
 src/inspect/unnamed_namespace_check.hpp:29:  Weird number of spaces at line-start.  Are you using a 2-space indent?  [whitespace/indent] [3]
-src/inspect/unnamed_namespace_check.hpp:30:  Do not indent within a namespace.  [whitespace/indent_namespace] [4]
```

Comment:
This is technically a false negative, but it's due to the already-violation that is whitespace/braces so I don't think it's worth the extra compute tradeoff to support.